dash/dash.py

```diff
@@ -1098,8 +1100,9 @@ def _setup_server(self):
     def _add_assets_resource(self, url_path, file_path):
         res = {"asset_path": url_path, "filepath": file_path}
         if self.config.assets_external_path:
-            res["external_url"] = "{}{}".format(
-                self.config.assets_external_path, url_path
+            res["external_url"] = "/".join([
+                self.config.assets_external_path.rstrip('/'), self.config.assets_url_path.strip('/'), url_path.lstrip('/')
+            ]
```

Comment:
Hmm, maybe we could just call `get_asset_url` here?

Reply:
That works! Passing all tests.